src/libs/browser/webview.cpp

```diff
@@ -227,7 +227,7 @@ void WebView::mouseReleaseEvent(QMouseEvent *event)
                         .arg(clickedLink.toString()));
 
 
-            QCheckBox *checkBox = new QCheckBox("Do &not ask again");
+            auto *checkBox = new QCheckBox("Do &not ask again");
```

Comment:
In this case there is no point in `auto *`, and it should simply be `auto`. Same applies to similar situations below.